dist/colors.color-map.scss

```diff
@@ -1,84 +1,84 @@
 $polaris-colors: (
   'purple': (
-    'lighter': rgb(246, 240, 253),
-    'light': rgb(227, 208, 255),
-    'base': rgb(156, 106, 222),
-    'dark': rgb(80, 36, 143),
-    'darker': rgb(35, 0, 81),
     'text': rgb(80, 73, 90),
+    'darker': rgb(35, 0, 81),
+    'dark': rgb(80, 36, 143),
+    'base': rgb(156, 106, 222),
+    'light': rgb(227, 208, 255),
+    'lighter': rgb(245, 240, 252),
```

Comment:
purple-lighter has changed, is that normal?

Before / after:

```diff
- rgb(246, 240, 253)
+ rgb(245, 240, 252)
```

Reply:
When we looked at the diff together there were values that had changed so I updated them in this round back to the previous values that were in this repo. So they have changed but it's just correcting them back to what they were.

Reply:
Some colors are still off. This also means that people using DSM (if there are any) are using the wrong colors at this time.

Here's the baseline that DSM should follow: https://shopify.github.io/polaris-tokens/

Hope this helps!